QMMLoader/QMMLoader.cs

```diff
@@ -0,0 +1,61 @@
+﻿using BepInEx;
+using BepInEx.Logging;
+using HarmonyLib;
+using System.Reflection;
+
+namespace QMMLoader
```

Comment:
```suggestion
namespace QModManager.QMMLoader
```
I think we should keep everything (both private and public) under the `QModManager` namespace.